gapidapk/deviceinfo.go

```diff
@@ -108,6 +108,7 @@ func fetchDeviceInfo(ctx context.Context, d adb.Device) error {
 
 	packages := []string{}
 	driver, err := d.GraphicsDriver(ctx)
+	gpuProfilingSupported := false
```

Comment:
This block of code is used in two places I think, which I plan to clean up. So maybe Hugues you could step in to lift this block into a function? Sorry for adding more work, otherwise I can accept it as a temporary workaround to unblock people

Reply:
I can have a go at this tomorrow.